router-tests/websocket_test.go

```diff
@@ -2411,19 +2524,34 @@ func handleCountEmpSubscription(t *testing.T, wsWriteCh chan<- wsMessage, id str
 		time.Sleep(updateInterval)
 	}
 
-	// Send complete message
-	t.Log("Sending complete message for subscription")
-	done := make(chan error, 1)
-	wsWriteCh <- wsMessage{
-		data: GraphQLWSDataResponse{
-			Type: "complete",
-			ID:   id,
-		},
-		done: done,
-	}
-	if err := <-done; err != nil {
-		t.Logf("Failed to send complete message: %v", err)
+	if complete {
+		// Send complete message
+		t.Log("Sending complete message for subscription")
```

Comment:
should we remove log statements to keep test logs clean?

Reply:
> For tests, the text will be printed only if the test fails or the -test.v flag is set. For benchmarks, the text is always printed to avoid having performance depend on the value of the -test.v flag.

Reply:
if it logs only on fail, it's fine